entrypoints/utils/setup-hocus-dev-env.ts

```diff
@@ -98,9 +98,11 @@ export async function setupHocusDevEnv() {
   const agentConfig = injector.resolve(Token.Config).agent();
   const db = new PrismaClient({ datasources: { db: { url: agentConfig.databaseUrl } } });
 
-  const existingDevUser = await db.user.findUnique({ where: { externalId: DEV_USER_EXTERNAL_ID } });
-  if (existingDevUser) {
-    console.log("Dev user already exists, skipping setup");
+  const existingDevRepository = await db.gitRepository.findUnique({
```

Comment:
Nuuuuu this will break my setup again. I redeploy Hocus very often:
```
AGENT_DEV_CREATE_HOCUS_PROJECTS=true HOCUS_HOSTNAME=setup1.hedgehog-hen.ts.net ./ops/bin/local-up.sh
```
I don't create the test repo when deploying Hocus